Dna_Rna.py

```diff
@@ -12,17 +16,33 @@ def reverse_complement(self):
 
     def gc(self):
         gc_amount = (self.sequence.count("C") + self.sequence.count("G")) / len(self.sequence)
-        return gc_amount
+        return round(gc_amount, 2)
 
     def __repr__(self):
         return self.sequence
 
 
-class Dna(Rna):
-    def __init__(self, seq):
-        self.dna_seq = seq
+class Dna:
+    def __init__(self, sequence):
+        self.sequence = sequence.upper()
+        self.complement = ""
+
+        if 'A' or 'C' or 'G' or 'T' in self.sequence:
+            pass
+        else:
+            raise 'This is not a Dna'
 
     def transcribe(self):
-        rna_seq = self.dna_seq.replace('T', 'U')
-        rna_complement = Rna(rna_seq).reverse_complement()
+        rna_complement = Rna(self.sequence.replace('T', 'U'))
         return rna_complement
+
+    def gc(self):
+        gc_amount = (self.sequence.count("C") + self.sequence.count("G")) / len(self.sequence)
+        return round(gc_amount, 2)
```

Comment:
Maybe return value in percents? That's not obligatory, just suggestion.

Reply:
Do you mean adding special sign in the end?

Reply:
I meant that now the output of gc() is 0.5, and  it seems me more convenient when instead of 0.5 we receive 50%